### ISSUE TITLE
Fix Report Abuse Page's Accessibility

### ISSUE THREAD

**Issue by loic-sharma** [NuGet/NuGetGallery#4002]
The Report Abuse page's help texts should be marked as accessibility labels for their respective inputs.
